pkg/csi/manila/options/shareoptions.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"k8s.io/cloud-provider-openstack/pkg/share/manila/shareoptions/validator"
+)
+
+type ControllerVolumeContext struct {
+	Protocol       string `name:"protocol" matches:"^(?i)CEPHFS|NFS$"`
+	Type           string `name:"type" value:"default:default"`
+	ShareNetworkID string `name:"shareNetworkID" value:"optional"`
+
+	// Adapter options
+
+	CephfsMounter  string `name:"cephfs-mounter" value:"default:fuse" matches:"^kernel|fuse$"`
+	NFSShareClient string `name:"nfs-shareClient" value:"default:0.0.0.0"`
```

Comment:
0.0.0.0/0

Reply:
There were issues with Ganesha NFS server, see e.g.:
https://bugs.launchpad.net/manila/+bug/1800627

And for some reason 0.0.0.0 worked. So this is where this oddity comes from. If this is not the case any longer then I agree the /0 should be there.

Reply:
This ganesha oddity was https://bugs.launchpad.net/manila/+bug/1800627 which we addressed by having the manila driver accept the conventional 0.0.0.0/0 but present (and recognize) 0.0.0.0 without the slash-zero when interacting with the ganesha service itself.  Fix was released and backported.

Reply:
Will do + i'll try to write some unit tests, probably next week or so though...completely swamped with school...

Reply:
Sorry for the hold up, one more busy week

Reply:
Done in https://github.com/kubernetes/cloud-provider-openstack/pull/536/commits/73bea65a8f7a0295980b526b7089b0c24c8badf3